diff --git a/node_modules/@badatgil/expo-mapbox-navigation/ios/ExpoMapboxNavigation.podspec b/node_modules/@badatgil/expo-mapbox-navigation/ios/ExpoMapboxNavigation.podspec
index 2ec9d73..fd30de1 100644
--- a/node_modules/@badatgil/expo-mapbox-navigation/ios/ExpoMapboxNavigation.podspec
+++ b/node_modules/@badatgil/expo-mapbox-navigation/ios/ExpoMapboxNavigation.podspec
@@ -10,7 +10,7 @@ Pod::Spec.new do |s|
   s.license        = package['license']
   s.author         = package['author']
   s.homepage       = package['homepage']
-  s.platforms      = { :ios => '13.4', :tvos => '13.4' }
+  s.platforms      = { :ios => '15.1', :tvos => '15.1' }
   s.swift_version  = '5.9'
   s.source         = { git: 'https://github.com/badatgil/expo-mapbox-navigation' }
   s.static_framework = true
@@ -44,3 +44,4 @@ Pod::Spec.new do |s|
   }
 
 end
+
